integration_tests/ctesque/src/sharedTest/java/android/content/res/ResourcesTest.java

```diff
@@ -647,9 +647,10 @@ public void openRawResource_withNonFile_throwsNotFoundException() {
   }
 
   @Test
-  @Ignore("todo: incorrect behavior on robolectric vs framework?")
-  public void openRawResourceFd_returnsNull_todo_FIX() {
-    assertThat(resources.openRawResourceFd(R.raw.raw_resource)).isNull();
+  public void openRawResourceFd_withNonCompressedFile_returnsNotNull() throws IOException {
+    try (AssetFileDescriptor afd = resources.openRawResourceFd(R.raw.raw_resource)) {
```

Comment:
It's better to add `assumeFalse(useLegacy())` because it also runs with Robolectric. Although I don't like add Robolectric APIs calling in sharedTest files.